package/MDAnalysis/analysis/encore/confdistmatrix.py

```diff
@@ -0,0 +1,339 @@
+        while i * (i - 1) / 2 < sum(runs_per_worker[:n + 1]):
+            i += 1
+        b = [i - 2,
+             sum(runs_per_worker[0:n + 1]) - (i - 2) * (i - 1) / 2 - 1]
+        tasks_per_worker.append((tuple(a), tuple(b)))
+        if b[0] == b[1]:
+            a[0] = b[0] + 1
+            a[1] = 0
+        else:
+            a[0] = b[0]
+            a[1] = b[1] + 1
+
+    # Allocate for output matrix
+    distmat = Array(c_float, matsize)
+
+    # Prepare progress bar stuff and run it
+    pbar = AnimatedProgressBar(end=matsize, width=80)
+    partial_counters = [RawValue('i', 0) for i in range(ncores)]
+
+    # Initialize workers. Simple worker doesn't perform fitting,
+    # fitter worker does.
+    
+    workers = [Process(target=conf_dist_function, args=(
+        tasks_per_worker[i],
+        rmsd_coordinates,
+        distmat,
+        masses,
+        fitting_coordinates,
+        subset_masses,
+        partial_counters[i],
+        args,
+        kwargs)) for i in range(ncores)]
+
+    # Start & join the workers
+    for w in workers:
+        w.start()
+    for w in workers:
+        w.join()
+    
+    # When the workers have finished, return a TriangularMatrix object
+    return TriangularMatrix(distmat, metadata=metadata)
+
+
+def set_rmsd_matrix_elements(tasks, coords, rmsdmat, masses, fit_coords=None,
+                       fit_masses=None, pbar_counter=None, *args, **kwargs):
+
+    '''
+    RMSD Matrix calculator
+
+    Parameters
+    ----------
+
+        tasks : iterator of int of length 2
+    Given a triangular matrix, this worker will calculate RMSD
+    values from element tasks[0] to tasks[1]. Since the matrix
+    is triangular, the trm_indeces matrix automatically
+    calculates the corrisponding i,j matrix indices.
+    The matrix is written as an array in a row-major
+    order (see the TriangularMatrix class for details).
+
+    If fit_coords and fit_masses are specified, the structures
+    will be superimposed before calculating RMSD, and fit_coords and fit_masses
+    will be used to place both structures at their center of mass and 
+    compute the rotation matrix. In this case, both fit_coords and fit_masses
+    must be specified.
+
+    coords : numpy.array
+        Array of the ensemble coordinates
+
+    masses : numpy.array
+        Array of atomic masses, having the same order as the
+        coordinates array
+
+    rmsdmat : encore.utils.TriangularMatrix
+        Memory-shared triangular matrix object
+
+    fit_coords : numpy.array or None
+        Array of the coordinates used for fitting
+
+    fit_masses : numpy.array
+        Array of atomic masses, having the same order as the
+        fit_coords array
+
+    pbar_counter : multiprocessing.RawValue
+        Thread-safe shared value. This counter is updated at
+        every cycle and used to evaluate the progress of
+        each worker in a parallel calculation.
+        '''
+
+    print coords
```

Comment:
These `print` statements need to go or if absolutely necessary, use `logger.debug()`.